.github/workflows/pull_request.yml

```diff
@@ -42,6 +50,25 @@ jobs:
       - name: Install Node.js packages
         if: ${{ steps.modules-cache.outputs.cache-hit != 'true' }}
         run: yarn --frozen-lockfile
+      - name: Set up git config
+        if: ${{github.actor == 'dependabot[bot]' }}
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update manifest
+        if: ${{github.actor == 'dependabot[bot]' }}
+        run: node scripts/createManifest.js
+      - name: Push koku-manifest changes
+        if: ${{github.actor == 'dependabot[bot]' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git add .
+          git commit --message "Update koku-manifest file"
+          git push
```

Comment:
If we push to the same PR, will this build exit? I'm assuming that this PR update generates a second build?

Reply:
it looks like the actions only ran once on this PR (didn't run on the koku-manifest commit) https://github.com/gitdallas/koku-ui/pull/6